client/components/screen-options-tab/style.scss

```diff
@@ -88,7 +88,7 @@ $screen-options-icon-border-y: 7px;
 	// When the user isn't hover over one of the buttons let's highlight the first
 	// because it is the Calypso option and its safe to assume they're in Calypso
 	// if they are seeing this component.
-	.screen-switcher__button:first-of-type {
+	.screen-switcher__button:first-child {
```

Comment:
Just curious for my own understanding why we updated these?

Reply:
Because the children are no longer the same type (first child is a button, and 2nd child is an anchor).